stl/inc/xstring

```diff
@@ -2369,6 +2369,13 @@ private:
     static constexpr size_type _ALLOC_MASK            = _Scary_val::_ALLOC_MASK;
     static constexpr size_type _SMALL_STRING_CAPACITY = _Scary_val::_SMALL_STRING_CAPACITY;
 
+    _NODISCARD static _CONSTEXPR20 bool _Entails_large_string(size_type _Size_or_capacity) noexcept {
+        // Use cases:
+        // 1. determine whether large mode is used for a given capacity.

```

Comment:
As case 1 is not used often and shares exactly the same intention as `_Large_string_engaged`(all replaced in [this commit](https://github.com/microsoft/STL/pull/3862/commits/8ca3f1c499a9725336ebc039a7a478556cb63e7d)), I think we'd better add comments like `// large string engaged` for them.

Reply:
This is moot after Casey expanded this function at each callsite.